src/content/en/2020/caching.md

```diff
@@ -94,6 +94,14 @@ When considering what to cache, it is important to understand whether the respon
 * An example of static content is an image.  For instance, a picture of a cat is the same regardless of who's requesting it or where the requester is located.
 * An example of dynamic content is a list of events which are specific to a geographic location. The list will be different based on the requester's location.
 
+{{ figure_markup(
+  image="luna-cat.png",
+  link="https://almanac.httparchive.org/static/images/2020/caching/luna-cat.png",
```

Comment:
The link happens automatically so no need to include this. It's meant for if you want a different link for some reason.

```suggestion
```